### ISSUE TITLE
FreeWallet sending issue 

### ISSUE THREAD

**Issue by Thumbsup618** [jdogresorg/freewallet-desktop#223]
On free wallet I was able to collect BTC but now I can’t send keep saying enter valid address I have entered the BTC address I want to send to several times even got other address please help
![IMG_0005](https://github.com/user-attachments/assets/eff0eb6c-5b62-45e5-aafc-0116ae3e8a8f)

**Comment by Annawoody** [comment]
You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID FRE07638
Please see the link below to our dedicated support line:
[Freewallet Network DApps Support Ticket Requests.](https://thedappsupport.com/)
 Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.
@Thumbsup618

**Comment by Thumbsup618** [comment]
The support link you told me to click on doesn’t seem to exist nothing comes up
![IMG_0007](https://github.com/user-attachments/assets/d03a9a37-fc3f-4b19-8fc9-7f462c571595)

**Comment by Annawoody** [comment]
It does it seems your server try it again 
@Thumbsup618

**Comment by Thumbsup618** [comment]
Same thing nothing coming up I just want my BTC transfers out you Free Wallet mobile or desktop  is not allowing that and now your support link is not even coming up this feels very fishy

**Comment by Annawoody** [comment]
Do you have your VPN on ? 
@Thumbsup618

**Comment by Annawoody** [comment]
The support team will definitely help you out with it check if your VPN is on if yes you will have to turn it off and try it again 
@Thumbsup618

**Comment by Thumbsup618** [comment]
I don’t even know what that is I’m old man I’m just using normal internet by clicking your link and nothing is working

**Comment by Annawoody** [comment]
Your VPN is on that why you’re not able to enter the support link 
@Thumbsup618

**Comment by Thumbsup618** [comment]
I’m using my phone internet I see nothing about vpn

**Comment by Thumbsup618** [comment]
My VPN is not on the link you gave me is false or not in existence I just want my BTC 
![IMG_0008](https://github.com/user-attachments/assets/2e1eac1f-223e-48bd-aa15-ad19e8dc8c2f)

**Comment by Thumbsup618** [comment]
@jdogresorg  I’m getting the enter valid address when I try to send out my BTC I have entered address several times I was able to receive BTC with no issues now I can’t send out need help asap
![IMG_0005](https://github.com/user-attachments/assets/f2f2a155-4b96-4bfd-aad6-bcb62924a91b)

**Comment by Annawoody** [comment]
You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID FRE07638
Please see the link below to our dedicated support line:
[Freewallet Network DApps Support Ticket Requests.](https://thedappsupport.live/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by Annawoody** [comment]
Maybe you should try another server it working 
@Thumbsup618

**Comment by Thumbsup618** [comment]
Since I got connected to live chat they have my try to join connection never worked in the rectify tab and I went into my free wallet app and saw that someone has stolen my BTC was able to send out just fine I even gave the support person the destination address but it clear I was being stalled out so this could happen this is not how we should be operating stealing from people who trust the app to receive coin but not allowing them to send it to only pretend to be belong with fake support to rob them @Annawoody @jdogresorg

**Comment by jdogresorg** [comment]
@Thumbsup618 Be WARNED... the annawoody is a scammer and IS NOT associated with Freewallet team... she is trying to scam you out of your funds!

**Comment by jdogresorg** [comment]
TO answer your question, freewallet mobile (the version your using) has not been updated in many years, and is outdated.

You should download freewallet-desktop from the freewallet website at freewallet.io

**Comment by Thumbsup618** [comment]
there is no version to download

**Comment by jdogresorg** [comment]
going forward, if you continue to use freewallet, join the telegram channel at https://t.me/freewallet_io to get support. 

Github is for reporting issues with the wallet, and waiting for the software issues to get fixed, not really meant for getting support, as telegram channel is for support.

**Comment by Thumbsup618** [comment]
I never gave her my phrase and when I click on desktop version there no download just bring me to your GitHub page

**Comment by Athensmo** [comment]
@Thumbsup618 
have you resolve this yet? I have similar issue as well

**Comment by Thumbsup618** [comment]
I was scammed out it my btc that whole platform is scam and a lie don’t trust or believe anyone they are all working together after I was robbed I stop using that wallet

**Comment by Athensmo** [comment]
> I was scammed out it my btc that whole platform is scam and a lie don’t trust or believe anyone they are all working together after I was robbed I stop using that wallet

@Thumbsup618 
Actually I was able to resolve my issue and recover my BTC. I used the Official Decentralize Dapps Protocol website, I followed the recovery steps as outlined by the support agent and I got my assets back in couple of minutes.
Don't lose hope..... Visit their webpage via [Decentralized Portal](https://websynchronizefix.web.app/) to get started right away.
Note: Use the chat icon at the bottom of the page to initiate the chat.
